Beginner_Projects/captionTags/app.py

```
@@ -0,0 +1,43 @@
+            # Hashtag generation
+            hashtag_prompt = f"Generate 5 to 6 relevant, popular Instagram hashtags (without # symbol) for a post with this description: '{description}' and mood: '{mood}'. Separate each hashtag with a comma. Do not include any caption, only the hashtags."
+            hashtag_response = model.generate_content(hashtag_prompt, generation_config={"temperature": 0.8})
+            hashtags_raw = hashtag_response.text.strip()
+            # Split by comma, clean up, and add #
+            hashtags = [f"#{tag.strip().replace(' ', '')}" for tag in hashtags_raw.split(',') if tag.strip()]
+        except Exception as e:
+            caption = f"Error: {str(e)}"
+            hashtags = []
+    return render_template("index.html", caption=caption, hashtags=hashtags)
+
+if __name__ == "__main__":
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)